src/main/java/com/jcabi/github/Hooks.java

```diff
@@ -81,13 +81,16 @@
      * Create new hook.
      * @param name Hook name
      * @param config Configuration for the hook
+     * @param active Actually trigger the hook when the events occur?
      * @return Hook
      * @throws IOException If there is any I/O problem
      * @see <a href="http://developer.github.com/v3/repos/hooks/#create-a-hook">Create a hook</a>
+     * @todo #1106:30min Add support for the "events" parameter.
```

Comment:
@cvrebert this puzzle description is too short. the minimum length is 20 words - please expand a little bit.